cmd/configschema/fields.go

```diff
@@ -21,8 +21,27 @@ import (
 	"time"
 
 	"github.com/fatih/structtag"
+	"go.uber.org/multierr"
 )
 
+func ConfigsToFields(dr DirResolver, configs []CfgInfo) (map[string]map[string]*Field, error) {
```

Comment:
Is this function being used?

Reply:
Apparently it's vestigial at this point. Will remove.